package.json

```diff
@@ -39,7 +39,7 @@
     "enzyme": "^3.11.0",
     "enzyme-to-json": "^3.2.2",
     "eslint": "^6.8.0",
-    "eslint-config-amex": "^11.2.0",
+    "eslint-config-amex": "^12.2.0",
```

Comment:
I am not a fan of `props` to `properties`, I believe that this lint rule is removed in later versions.

Reply:
Ah! Though that's what "props" is short for I can add an ignore directive before the update to v13 👍